docs/howto.rst

```diff
@@ -1,23 +1,141 @@
 How-to...
 =========
 
-Ensure tasks don't run concurrently and run in order
-----------------------------------------------------
+Ensure jobs don't run concurrently and run in order
+---------------------------------------------------
 
-Launch a task in the future
+Let's imagine we have a task like this::
+
+    @task_manager.task(queue="queue")
+    def write_alphabet(floor):
+        time.sleep(random.random() * 5)
+        with open("/tmp/alphabet.txt", "a") as f:
+            f.write(floor)
+
+We write the letter we recieve to a file after wating for a
+random time (this is a simplified version of a real
+world problem where tasks take a unforseeable amount of time
+and share resources like a database).
+
+We call it::
+
+    write_alphabet.defer("a")
+    write_alphabet.defer("b")
+    write_alphabet.defer("c")
+    write_alphabet.defer("d")
+
+We could expect the following to be written in `pyramid.txt`::
```

Comment:
```suggestion
We could expect the following to be written in `alphabet.txt`::
```

Reply:
Oh yeah right. I started with a different example, and simplified it :D